File: v2/src/main/kotlin/com/api/v2/persons/PersonResponseMapper.kt
package com.api.v2.persons

import com.api.v2.persons.domain.Person
import com.api.v2.persons.dtos.PersonResponseDto

class PersonResponseMapper {

    companion object {
        fun map(person: Person): PersonResponseDto {
            return PersonResponseDto(
                person.fullName(),
                person.birthDate,
                person.ssn,
                person.email,
                person.address,
                person.gender,
                person.phoneNumber
            )
        }
    }

}

File: c#/V4/Persons/Utils/PersonResponseMapper.cs
using V4.Persons.Domain;
using V4.Persons.DTOs;

namespace V4.Persons.Utils;

public class PersonResponseMapper
{

    public static PersonResponseDto Map(Person person)
    {
        return new PersonResponseDto(
            person.FullName(),
            person.DateOfBirth,
            person.Ssn,
            person.Email,
            person.PhoneNumber,
            person.Address,
            person.Gender
        );
    }

}